src/cromshell/utilities/VariableSetup.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+
+def get_submission_file(config_dir):
+    submission_file = "{}/all.workflow.database.tsv".format(config_dir)
+    if not Path.exists(Path(submission_file)):
+        Path.touch(Path(submission_file))
+        submission_header = 'DATE{0}CROMWELL_SERVER{0}RUN_ID{0}WDL_NAME{0}STATUS{0}ALIAS'.format('\t')
+        with submission_file.open() as f:
+            f.write_text(submission_header)
+    return submission_file
+
+
+def get_config_dir():
+    config_path = str(Path.home()) + "/.cromshell"
+    Path.mkdir(Path(config_path), exist_ok=True)
+    return config_path
+
+
+class Cromwell:
+    server = "https://cromwell-v47.dsde-methods.broadinstitute.org/"
+    verbosity = "q"
```

Comment:
There is already a verbosity argument in the main file, i'd like to use it to set this argument too.